### ISSUE TITLE
remove hidden attribute from aria-live elements

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#544]
It is not necessary, and only forces clients to explicitly unhide elements when they want an accessible alert. When we created the group of aria-live elements initially, we hoped that hiding the group of elements would prevent NVDA from reading "alert" on sim startup. NVDA reads this regardless, so the hidden attribute should be removed.

**Comment by jessegreenberg** [comment]
Done, closing.
